### PR TITLE
Final tweaks to database seed

### PR DESCRIPTION
**Pull request by leftclickben**
Fixes RWAHS-603 and RWAHS-611.

### CODE REVIEW

**Comment by leftclickben** [comment]
Assigned to both @kehh and @mostro20 to fight over ;-)

File: db/seed.sql [hunk 1]
```diff
@@ -4069,6 +4071,7 @@ INSERT INTO `cms_page` VALUES (5,'Copyright','1column',NULL,NULL,'copyright','Co
...
...;&nbsp;&nbsp;&nbsp; Copyright is the responsibility of the client.&nbsp; If the Society does not hold copyright ownership, the client will be required to provide a licence or written permission to reproduce from the copyright holder, prior to the supply of photographic material.&nbsp; The client will be required to sign an indemnification as part of their order form.</p>\r\n<p>10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The type of publication must be clearly indicated:&nbsp; eg general book, artbook illustration, CD cover, text book, brochure, calendar, poster, exhibition, exhibition catalogue, reproduction print, CD rom, television, web page.</p>\r\n<p>11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Photographs must not be photocopied, as this process damages photographs by exposing them to heat, excessive light and ozone.</p>\r\n<p>12.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The Society reserves the right to refuse permission for reproduction of a photograph, artwork or other printed item if it considers that this is not in the best interests of the Society.</p>','2016-09-12 13:31:06','2016-09-12 13:37:29',1,0,NULL,NULL,NULL,NULL,NULL,NULL);
```
Comment (kehh) [inline db/seed.sql:4074]: Surprised you didn't use an `<ol>` but I'm not one to judge.

Comment (leftclickben) [reply db/seed.sql:4074]: Copied and pasted from page on production website. It's common with legal stuff to use preformatted text, not sure why, portability?

Comment (leftclickben) [reply db/seed.sql:4074]: Sorry I didn't annotate this PR.

**Pull request merged**